integration/kubernetes/helm-chart/alluxio/README.md

```diff
@@ -0,0 +1,82 @@
+# Alluxio
+
+[Alluxio](https://www.alluxio.io/) is a Data orchestration for analytics and machine learning in the cloud.
+
+
+## Introduction
+
+This chart bootstraps [Alluxio](https://www.alluxio.io/) cluster on a [Kubernetes]() cluster using the [Helm]() package manager.
+
+
+## Prerequisites
+
+* Kubernetes 1.11+ with Beta APIs enabled 
+
+
+## Install the Chart
+
+To install the Alluxio Chart into your Kubernetes cluster :
+
+```
+helm install --namespace "alluxio" --name "alluxio" alluxio
+```
+
+After installation succeeds, you can get a status of Chart
+
+```
+helm status "alluxio"
+```
+
+## Uninstall the Chart
+
+If you want to delete your Chart, use this command:
+
+```
+helm delete  --purge "alluxio"
+```
+
+## Configuration
```

Comment:
remove this table as it is easy to go stale? we can refer to https://docs.alluxio.io/os/user/edge/en/deploy/Running-Alluxio-On-Kubernetes.html instead as the single source of truth (after adding the relevant section there)